### ISSUE TITLE
NVDA and problems with youtube

### ISSUE THREAD

**Issue by nvaccessAuto** [nvaccess/nvda#5042]
**Reported by adelcia on 2015-04-20 18:31**
When the upload videos, NVDA around so I read that, that during the recording I have to leave this page open. A reading even when the page Minimize next hours. But for example, in Google Chrome me when recording video NVDA does not read such stupidity.

**Comment by bhavyashah** [comment]
I was uploading a couple YouTube videos a few months ago. If my memory serves me right, I remember encountering no such issue on Firefox. I cannot speak for Chrome, since I don't recall performing this there. Since it is likely that the YouTube video uploading interface has significantly changed since this report and since we have no means of reaching the original author of this ticket to request retesting, unless any NVDA user+YouTuber can confirm this ticket, I suggest closing. @ehollig

**Comment by ehollig** [comment]
The original poster is stating the issue is in Firefox with youtube, and @bhavyashah states the problem did not exist in https://github.com/nvaccess/nvda/issues/5042#issuecomment-323510997. Closing, unless someone else experiences something different using latest Firefox and NVDA.
